snapcraft/storeapi/_upload.py

```diff
@@ -48,7 +54,8 @@ def upload(binary_filename, metadata_filename='', metadata=None, config=None):
         return
     name = match.groupdict()['name']
 
-    logger.info('Uploading files...')
+    logger.info('')
```

Comment:
Why did you keep this one? I might be missing something, but I fail to see a reason for logging nothing :)